nearlib/package.json

```diff
@@ -26,7 +27,7 @@
     "test": "jest test",
     "lint": "eslint .",
     "fix": "eslint . --fix",
-    "doc": "documentation readme near.js signing/key_pair.js -f md --shallow --section=API"
+    "doc": "documentation readme near.js account.js signing/key_pair.js -f md --shallow --section=API --readme-file=API.md"
```

Comment:
should we just do wildcards like `*.js signing/*.js`, etc?

Reply:
This tool generates something pretty crappy if the file is not properly js-docced, so it's better for now to have an explicit list of files. We can easily swith to *js in the future